lib/tools/adb-commands.js

```diff
@@ -1151,7 +1154,14 @@ methods.killProcessByPID = async function killProcessByPID (pid) {
       log.info(`Cannot kill PID ${pid} due to insufficient permissions. Retrying as root`);
       let {isSuccessful} = await this.root();
       becameRoot = isSuccessful;
```

Comment:
It's outside of your change, but this can be `({isSuccessful: becameRoot} = await this.root());` and avoid the throw-away variable.

Reply:
thanks for the suggestion